nbbuild/cluster.properties

```diff
@@ -660,6 +660,7 @@ nb.cluster.java=\
         lib.nbjshell,\
         lib.nbjshell9,\
         libs.cglib,\
+        libs.corba.omgapi,\
```

Comment:
New Wrapper module to add  ext jar glassfish-corba-omgapi-4.1.0-jar as project dependency